src/main/java/com/capitalone/dashboard/model/LibraryPolicyResult.java

```diff
@@ -303,10 +303,6 @@ public void setPolicyAlert(List<PolicyScanMetric> policyAlert) {
         this.policyAlert = policyAlert;
     }
 
-    private String getComponentPlusDisposition (String component, LibraryPolicyThreatDisposition disposition) {
```

Comment:
i am assuming this is unused private method, please confirm

Reply:
Yes this is unused unless through some complex reflection. I did a string search as well through the repository. However this method Kinda looks like it is part of a group. I might not have removed it if I were making the changes (I did not personally do this one), so I would be happy to leave it in (revert) if you would prefer that.